symplyphysics/laws/thermodynamics/isobaric_potential_from_heat_capacity.py

```diff
@@ -0,0 +1,58 @@
+from sympy import (Eq, solve, log)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## The isobaric potential of a reaction is a value whose change during a chemical reaction is equal to the change in the internal
+## energy of the system. The isobaric potential shows how much of the total internal energy of the system can be used for chemical
+## transformations.
+## Thermal effect of reaction is change of enthalpy of the system.
+
+## Law is: G = H - T * S - Cp * T * (ln(T / 298) + (298 / T) - 1), where
+## G - change of isobaric potential of reaction,
+## H - thermal effect of reaction,
+## T - temperature,
+## S - change of entropy,
+## Cp - change of heat capacity.
```

Comment:
It seems it is measured in standard state. You can add small explanation and `standard` to symbols, where it's appropriate.